client/components/Entry/EntryPage.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <main>
+        <section v-if="$store.state.username">
+            <button class="btn-secondary">
+                <router-link to="/newLog">Add Log ➕</router-link>
+            </button>
+
+            <table class="entries" v-if="filteredEntries">
+                <!-- <tr>
+                    <th>Type</th>
+                    <th>Title</th>
+                    <th>Date</th>
+                </tr> -->
+                <EntryComponent 
+                    v-for="entry in filteredEntries" 
+                    :key="entry._id"
+                    :entry="entry"
+                />
+            </table>
+        </section>
+        <section v-else>
+            <h2>Please log in to your account first!</h2>
+        </section>
+    </main>
+</template>
+
+<script>
+import EntryComponent from '@/components/Entry/EntryComponent.vue';
+
+export default {
+    name: 'EntryPage',
+    components: {EntryComponent},
+    props: {
+        type: {
+            type: String,
+            required: true
+      }
+    },
+    computed: {
+        activeType: function() {
+            const type = this.type;
+            return type.charAt(0).toUpperCase() + type.slice(1);
+        },
+        filteredEntries: function() {
+            const entries = this.$store.state.entries;
+            if (this.type === "medication") {
+                return entries.filter((entry) => {return entry.type === "medication"});
+            } else if (this.type === "appointment") {
+                return entries.filter((entry) => {return entry.type === "appointment"});
+            } else if (this.type === "other") {
+                return entries.filter((entry) => {return entry.type === "other"});
```

Comment:
Also got rid of ts here since declaring `entry` to be of a generic `Object` introduced compilation errors when accessing `entry.type`, not ideal. Alternatively, instead of making `filteredEntries` a computed value, do some `v-if` logic? But probably not great to wrap `v-if` inside a `v-for` either...